### ISSUE TITLE
Importing dynamic fields is not supported

### ISSUE THREAD

**Issue by fufenghua** [zilliztech/attu#544]
**Describe the bug:**
![image](https://github.com/zilliztech/attu/assets/18100287/daaaf5cb-a7cf-4b97-8c9e-00082cd01a84)
![image](https://github.com/zilliztech/attu/assets/18100287/caeb2a48-3a2a-4e22-ae43-c03efd669e18)
![image](https://github.com/zilliztech/attu/assets/18100287/a88b6439-dff0-43f9-b9e5-0661d3103862)
![image](https://github.com/zilliztech/attu/assets/18100287/3639f259-f4a7-4108-b041-f60ec423a3a6)
![image](https://github.com/zilliztech/attu/assets/18100287/77772b5b-2619-4ed9-9130-b7a3bda6efdd)

**Steps to reproduce:**
1.Exporting data from attu
2.Select Import data from file
3.Prompt column number mismatch
4.If you delete the $mate column, it will work fine.

**Attu version:2.4.0

**Comment by shanghaikid** [comment]
You don't have to use $meta.  please check here to see the example data format.

![image](https://github.com/zilliztech/attu/assets/185051/6b9f4c79-b538-4db6-8a8e-09480018571e)

**Comment by fufenghua** [comment]
> You don't have to use $meta. please check here to see the example data format.
> 
> ![image](https://private-user-images.githubusercontent.com/185051/341260612-6b9f4c79-b538-4db6-8a8e-09480018571e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5NTc1MTgsIm5iZiI6MTcxODk1NzIxOCwicGF0aCI6Ii8xODUwNTEvMzQxMjYwNjEyLTZiOWY0Yzc5LWI1MzgtNGRiNi04YThlLTA5NDgwMDE4NTcxZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyMVQwODA2NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zOGE0YmQ1MzYwNTAzNDdkYmY0NWExN2FlOTU0YTJmNDU4ZGEzM2MzYWY3ZjE1MjdlZTQwNjhlZDZjMzM1ZjY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.-h2nw5w3FKJtKKafQmVQj61bq-6iOK6aJ20xn6Fq2pM)

After downloading the sample data file, it still cannot be imported。
![image](https://github.com/zilliztech/attu/assets/18100287/b1a7a737-d940-4568-a4e6-b525d9db83f0)
![image](https://github.com/zilliztech/attu/assets/18100287/5132206e-56f1-4190-a762-734c4c717cc8)

**Comment by shanghaikid** [comment]
yeah, looks like csv import is broken. please use json import now, I will fix it soon.  thanks.
